src/components/peoplePage.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect } from 'react';
+import { Person } from '../types';
+import { Loader } from './Loader';
+import { PersonElement } from './PersonElement';
+
+interface Props {
+  people: Person[],
+  isLoading: boolean,
+  loadingError: boolean,
+  setSelectedPerson: (value: string) => void,
+  selectedName: string | undefined,
+  setSelectedName: (value: string | undefined) => void,
+}
+
+export const People: React.FC<Props> = ({
+  people,
+  isLoading,
+  loadingError,
+  setSelectedPerson,
+  setSelectedName,
+  selectedName,
+}) => {
+  const handleSelection = (
+    name: string | undefined,
+    slug: string,
+  ) => {
+    localStorage.setItem('selectedPersonSlug', slug);
+    setSelectedName(name);
+    setSelectedPerson(slug);
+  };
+
+  const updatedPeople = people.map((child) => {
+    if (child.motherName === null && child.fatherName === null) {
```

Comment:
Use destructuring for the child and use logical operators 
```suggestion
    if (!motherName && !fatherName) {
```